### DISCUSSION TITLE
osu Freezes when switch apps in Android

### DISCUSSION THREAD

**Discussion by JediMadeThis** [ppy/osu#17057] [Q&A] [answered]
I'm using android to play osu lazer then when i switch to another app, and then switch back to it again, it freezes.

**Comment by frenzibyte** [comment, marked as answer]
https://github.com/ppy/osu-framework/issues/4994

**Reply by frenzibyte** [reply]
Also for reference, can you specify your mobile device in case that comes in handy?

**Reply by JediMadeThis** [reply]
Samsung galaxy S10 and Samsung Tab S7
